testing/regress/ecl/javaembed.ecl

```diff
@@ -27,10 +27,15 @@ string jcat(string a, string b) := EMBED(java)
 ENDEMBED;
 
 integer jadd(integer a, integer b) := EMBED(java)
+import com.HPCCSystems.HpccUtils;
```

Comment:
as an fyi we've been using org.hpccsystems as our namespace

Reply:
You may have, but the com.HPCCSystems namespace as used by the embedded Java code predates your usage, I suspect...

It's probably too late to unify the two usages now.